silk-plugins/silk-plugins-rdf/src/main/scala/org/silkframework/plugins/dataset/rdf/formatters/NTriplesQuadFormatter.scala

```diff
@@ -0,0 +1,46 @@
+package org.silkframework.plugins.dataset.rdf.formatters
+
+import org.silkframework.dataset.rdf.{Quad, _}
+
+class NTriplesQuadFormatter() extends QuadFormatter {
+
+  private def escape(value: String) = org.apache.jena.atlas.lib.StrUtils.escapeString(value)
+
+  private def format(quad: Quad, asQuad: Boolean = true): String = {
```

Comment:
Please reuse the methods from org.silkframework.plugins.dataset.rdf.RdfFormatUtil

Reply:
Even better, since RdfFormatUtil defaults back to RDFDataMgr I do the same directly